README.md

```diff
@@ -606,8 +616,17 @@ Databases
 By default, sandboxes will use local, ephemeral databases that are destroyed
 when the sandbox is reprovisioned. If you want to reuse databases, change the
 `INSTANCE_EPHEMERAL_DATABASES` setting to False, set up external mysql and mongo
-databases and update the `INSTANCE_MYSQL_URL` and `INSTANCE_MONGO_URL` settings
-to point to these databases.
+databases and create one `MySQLServer`/`MongoDBServer` for each external
+mysql/mongo database that you want to use. When creating a new instance,
+it will automatically be assigned an existing `MySQLServer` and an existing `MongoDBServer`.
+If there are no `MySQLServer` or `MongoDBServer` objects to choose from
+when creating a new instance, the instance manager will automatically
+create a single `MySQLServer` and a single `MongoDBServer` using the values
+of the `DEFAULT_INSTANCE_MYSQL_URL` and `DEFAULT_INSTANCE_MONGO_URL` settings.
+So, if you want to skip creating a `MySQLServer` and a `MongoDBServer` manually,
+just make sure to add (or update) these settings in your `.env` file
+once you have obtained hostname and authentication information (username, password)
+for your external mysql and mongo databases.
```

Comment:
@pomegranited Thanks for the suggestions, I incorporated most of them into the README.

> Could you rephrase this paragraph to iterate the recommended steps to:

> * Provision a new VM to host MySQL or MongoDB - I found instructions for deploying a new MySQL server, but nothing for deploying a new MongoDB instance. Do we have instructions anywhere?

I'm not sure if we have instructions for deploying a new MongoDB instance anywhere. But I don't want to tie the instructions in the instance manager README to a specific workflow for setting up MySQL and MongoDB servers, so I didn't specify how to do this in the updated instructions.